backend/sentenceTokenizer.py

```diff
@@ -0,0 +1,52 @@
+class Tokenizer:
+
+    def __init__(self, text):
+        self.text = text
+        import nltk
```

Comment:
I would put the imports before the class definition, unless it's a conditional import statement. See [PEP8](https://www.python.org/dev/peps/pep-0008/)